onchain/src/quests/interfaces.cairo

```diff
@@ -1,17 +1,19 @@
 use starknet::ContractAddress;
 
-#[derive(Drop, starknet::Event)]
-pub struct QuestClaimed {
```

Comment:
Did this have to be moved for visibility reasons? I'd prefer to keep this event outside the `PixelQuest` so it can be used in other quest types.

Reply:
@b-j-roberts oh I see, sure i will move it back !